n3c_owl_ingest/n3c_owl_ingest.py

```diff
@@ -83,73 +92,99 @@
     'ID': 'ID',
     'Label': 'A rdfs:label',
     'Type': 'TYPE',
-    'domain_id': 'A N3C:domain_id',
-    'vocabulary_id': 'A N3C:vocabulary_id',
-    'concept_class_id': 'A N3C:concept_class_id',
-    'standard_concept': 'A N3C:standard_concept',
-    'concept_code': 'A N3C:concept_code',
-    'valid_start_date': 'A N3C:valid_start_date',
-    'valid_end_date': 'A N3C:valid_end_date',
-    'invalid_reason': 'A N3C:invalid_reason',
+    'domain_id': 'A OMOP:domain_id',
+    'vocabulary_id': 'A OMOP:vocabulary_id',
+    'concept_class_id': 'A OMOP:concept_class_id',
+    'standard_concept': 'A OMOP:standard_concept',
+    'concept_code': 'A OMOP:concept_code',
+    'valid_start_date': 'A OMOP:valid_start_date',
+    'valid_end_date': 'A OMOP:valid_end_date',
+    'invalid_reason': 'A OMOP:invalid_reason',
+    'rdfs:subClassOf': 'SC % SPLIT=|',
+}
+REL_PRED_MAP = {
```

Comment:
I've made it so that both of these OMOP relationship types get the same predicate:
    'Subsumes': 'rdfs:subClassOf',
    'RxNorm inverse is a': 'rdfs:subClassOf',